### PR TITLE
CC-4770 update packages

### PR DESCRIPTION
**Pull request by rlease**
I've marked the few file changes that I made. The rest are due to running `npm i`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,21 +4,14 @@
...
     "cleanup": "rm ./lib/mixins.css ./lib/variables.css",
-    "postcssify":
-      "find ./lib -type f -name '*.css' -exec ./node_modules/.bin/postcss -c postcss.json -r {} \\; && npm run cleanup",
-    "build":
-      "rm -rf lib/**/ && npm run babel && cpx \"./src/components/**/*.styl\" ./lib && stylus -I src lib/**/*.styl -u autoprefixer-stylus",
-    "dev":
-      "webpack-dev-server --config webpack.config.js --progress --profile --colors --hot",
+    "postcssify": "find ./lib -type f -name '*.css' -exec ./node_modules/.bin/postcss -c postcss.json -r {} \\; && npm run cleanup",
+    "build": "rm -rf lib/**/ && npm run babel && cpx \"./src/components/**/*.styl\" ./lib && stylus -I src lib/**/*.styl -u autoprefixer-stylus",
+    "dev": "webpack-dev-server --progress --profile --colors --hot",
```
Comment (rlease) [inline package.json]: I removed the check for a config file here.

File: package.json [hunk 2]
```diff
@@ -28,12 +21,12 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "lodash.uniqueid": "^4.0.1",
-    "react": "16.2.0",
+    "react": "^16.2.1",
```
Comment (rlease) [inline package.json]: I updated `react` and `react-dom`

**Comment by shal-xu** [comment]
Why are there so many style changes?

**Comment by shal-xu** [comment]
Will we have separate tickets for updating FE/Admin version?

**Comment by rlease** [comment]
I'll make the tickets for updating FE and admin. As for the style changes, ¯\\\_(ツ)_/¯. I just ran `npm i` and those are the changes it spit out.

**Comment by rlease** [comment]
I should bump the version of this package while I'm at it.

**Comment by shal-xu** [comment]
Yes

**Comment by BetaGrumm** [comment]
I assume we have some auto linting/prettier running that cleaned those out. I checked them all and they were the same values as the standard css property, so they should all apply just fine.

**Pull request merged**